spec/index.html

```diff
@@ -225,30 +301,29 @@
           attribute boolean? expanded;
           attribute boolean? disabled;
           attribute InvalidType? invalid;
-          attribute boolean hasPopUp;
+          attribute boolean? hasPopUp;
+          attribute boolean? pressed;
 
           // Control values.
-          attribute DOMString? value;
+          attribute DOMString? valueText;
```

Comment:
At the F2F, we normalized all variants of value (besides rangeValue) to a single "value" property. What's the need for a valueText property if we already have value. Also, I don't think we can remove value because it would be required for different field types (date, tel, etc). valueText appears to be future-limiting for the sole purpose of matching ARIA.